### PR TITLE
New functions: add, sigmoid, logit, exp

### PR DESCRIPTION
**Pull request by piotr1212**
add: adds a constant to a list of series, easier than sumSeries(serie,
constantLine(5)) and add a constant to a list of series without summing them.

exp: datapoint to the power of e

logit and sigmoid: logistic functions

Sort transform functions alphabetically in the composer pulldown.

### CODE REVIEW

**Comment by piotr1212** [comment]
see https://github.com/graphite-project/graphite-web/issues/2465

File: webapp/graphite/render/functions.py [hunk 1]
```diff
@@ -1456,6 +1462,133 @@ def scaleToSeconds(requestContext, seriesList, seconds):
...
+
+    Example:
+
+    .. code-block:: none
+
+      &target=sigmoid(Server.instance01.threads.busy)
+      &target=sigmoid(Server.instance*.threads.busy)
+
+    """
+    log.info("hi")
```
Comment (DanCech) [inline webapp/graphite/render/functions.py]: remove ;)

Comment (piotr1212) [reply webapp/graphite/render/functions.py]: lol

**Comment by deniszh** [comment]
Sorry, was (and still is) on vacation, with limited internet. Thanks for the review, @DanCech !

**Pull request merged**